hevc-webrtc-04.nr

```diff
@@ -123,9 +123,14 @@ Support for the H.265 video codec is OPTIONAL for WebRTC browsers and non-browse
 Implementations supporting H.265 that conform to this specification MUST support
 receiving H.265 and MAY support sending H.265. 
 
-For the H.265 [H.265] codec, endpoints MUST support the payload formats
-defined in [RFC7798].  In addition, they MUST support Main Profile
+For the H.265 [H.265] codec, endpoints MUST support Main Profile
 Level 3.1 (level-id=93) and SHOULD support Main Profile Level 4 (level-id=120).
+Also, endpoints MUST support the payload formats defined in [RFC7798], with
+the exception of Temporal Scalability Control Information (TSCI), defined
+in [RFC7798] Section 4.5. Rather than communicating TSCI using the PACI
```

Comment:
Need to specify If depacketizer does see PACI, should it be ignore?

Reply:
Not sure we can say that for any PACI.  But we can say that if RTP header extensions containing TCSI are negotiated, that TSCI info in the PACI MUST be ignored. Would that work?